utilities/validations.py

```diff
@@ -107,3 +108,13 @@ def validate_date_range(**kwargs):
     elif (('end_date' and 'start_date' in kwargs) and
             kwargs['end_date'] < kwargs['start_date']):
         raise ValueError('Earlier date should be lower than later date')
+
+
+def validate_room_labels(room_labels):
```

Comment:
The validator should ensure that the labels provided are existing in the structure table. Remember that these labels are the different office structure levels, like Block A, 1st Floor, e.t.c which must have been already set before a room is created.

Reply:
Okay noted. Will do that

Reply:
> The validator should also ensure that the labels provided are existing in the structure table. Remember that these labels are the different office structure levels, like Block A, 1st Floor, e.t.c which must have been already set before a room is created. This is to ensure that you cannot create a room in a structure that has not yet been set up in your location.

Okay noted. Will do that ASAP